recipe/meta.yaml

```diff
@@ -1,35 +1,45 @@
 {% set name = "arviz" %}
-{% set version = "0.11.2" %}
+{% set version = "0.15.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a9d0eb32e84a0472aa78a488ba9b12b05e7be8c2c8fb34a1ba6286cc1254ee0d
+  sha256: 981cce0282bdf6f3b379255b95a440979f9a0ef0ae9dd88a54f763cf5b31484c
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  # skipping 3.8 because of missing xarray-einstats for 3.8
+  skip: True  # [py==3.8]
+  skip: True  # [py<=2.7]
```

Comment:
```suggestion
  skip: True  # [py<39]
```

Reply:
Just a simplification.

Reply:
I thought these kinds of restrictions that are coming from the upstream project had to be retained even though we don't support `py<39`